schema/material.json

```diff
@@ -47,6 +51,10 @@
         "icsdId": {
             "description": "Corresponding ICSD id of the material",
             "type": "integer"
+        },
+        "isMolecule": {
```

Comment:
Maybe `isNonPeriodic` or `isIsolated` to make it clearer, since we could face molecular crystal

Reply:
Maybe let's add a comment about the default lattice (box) assumed for the isolated molecule